browser/about_flags.cc

```diff
@@ -11,6 +11,7 @@
 #include "brave/browser/brave_features_internal_names.h"
 #include "brave/browser/ui/brave_ui_features.h"
 #include "brave/browser/ui/tabs/features.h"
+#include "brave/browser/ui/views/tabs/switches.h"
```

Comment:
this needs views buildflag guard as this file is also compiled with non desktop?

Reply:
Oops, you're right. Added !IS_ANDROID() so that it matches with guard below.